### DISCUSSION TITLE
how to make bot not avoid suicide jumps?

### DISCUSSION THREAD

**Discussion by Grammf** [PrismarineJS/mineflayer#2244] [Q&A]
im making a bot for pvp server, and bot have to jump from 84 y to 5 y. How do i make my bot do it?
(fall damage is disabled)

**Comment by WhoTho** [comment]
Use mineflayer-pathfinder to walk to the edge of whatever platform you are dropping from, and then make it walk forward for a bit. You can use
```js
function wait(ms) {
  return new Promise((resolve) => setTimeout(resolve, ms));
}

bot.setControlState("forward", true);
await wait(1000);
bot.setControlState("forward", false);
```

Hope this helps!

**Reply by Grammf** [reply]
thanks for help. I just used moves.allowFreeMotion = true and it helped
